queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/RequestCommands/PosReceiptCommand.cs

```diff
@@ -0,0 +1,243 @@
+        {
+            var journals = await _journalITRepository.GetAsync().ConfigureAwait(false);
+            if (journals.Where(x => x.cbReceiptReference.Equals(request.cbReceiptReference)).Any())
+            {
+                throw new CbReferenceExistsException(request.cbReceiptReference);
+            }
+
+            var queueIt = await _countrySpecificQueueRepository.GetQueueAsync(queue.ftQueueId).ConfigureAwait(false);
+
+            var receiptResponse = CreateReceiptResponse(queue, request, queueItem, queueIt.CashBoxIdentification, _countryBaseState);
+
+            if (request.IsMultiUseVoucherSale())
+            {
+                return await CreateNonFiscalRequestAsync(queueIt, queueItem, receiptResponse, request).ConfigureAwait(false);
+            }
+
+            FiscalReceiptResponse response;
+            if (request.IsVoid())
+            {
+                var fiscalReceiptRefund = await CreateRefundAsync(request).ConfigureAwait(false);
+                response = await _client.FiscalReceiptRefundAsync(fiscalReceiptRefund).ConfigureAwait(false);
+            }
+            else
+            {
+                var fiscalReceiptinvoice = CreateInvoice(request);
+                response = await _client.FiscalReceiptInvoiceAsync(fiscalReceiptinvoice).ConfigureAwait(false);
+            }
+            if (!response.Success)
+            {
+                if (response.SSCDErrorInfo.Type == SSCDErrorType.Connection && !isBeingResent)
+                {
+                    return await ProcessFailedReceiptRequest(_countryspecificSettings, queue, queueItem, request).ConfigureAwait(false);
+                }
+                else
+                {
+                    throw new SSCDErrorException(response.SSCDErrorInfo.Type, response.SSCDErrorInfo.Info);
+                }
+            }
+            else
+            {
+                receiptResponse.ftReceiptIdentification += $"{response.ReceiptNumber}";
+                receiptResponse.ftSignatures = _signatureItemFactoryDefault.CreatePosReceiptSignatures(response);
+                var journalIT = new ftJournalIT().FromResponse(queueIt, queueItem, new ScuResponse()
+                {
+                    DataJson = response.ReceiptDataJson,
+                    ftReceiptCase = request.ftReceiptCase,
+                    ReceiptDateTime = response.ReceiptDateTime,
+                    ReceiptNumber = response.ReceiptNumber,
+                    ZRepNumber = response.ZRepNumber
+                });
+                await _journalITRepository.InsertAsync(journalIT).ConfigureAwait(false);
+            }
+
+            return new RequestCommandResponse
+            {
+                ReceiptResponse = receiptResponse,
+                Signatures = receiptResponse.ftSignatures.ToList(),
+                ActionJournals = new List<ftActionJournal>()
+            };
```

Comment:
```suggestion
            var queueIt = await _countrySpecificQueueRepository.GetQueueAsync(queue.ftQueueId).ConfigureAwait(false);
            
            return new RequestCommandResponse
            {
                ReceiptResponse = CreateReceiptResponse(queue, request, queueItem, queueIt.CashBoxIdentification, _countryBaseState),
                Signatures = new(),
                ActionJournals = new()
            };
```

I think you can just return it like that and remove everything above 🤔